docs/snapshots.rst

```diff
@@ -91,3 +93,33 @@ they next poll our mutable directory.
 We cannot sign local Snapshots because we don't yet have the
 capability-string for the content. The capability-string is only known
 after we upload the content to Tahoe-LAFS.
+
+
+Signature Details
+-----------------
+
+As above, we upload the content and metadata as two distinct immutable
+capability. There is also the "mangled" name of the file. This mangled
+name will also point at the Snapshot from the Personal DMD.
+
+The signature scheme is to concatenate all three of the above
+(separated by newlines) and then produce a signature over it using the
+author's signing key (which is a `nacl.signing.SigningKey` from the
+PyNaCl library (which uses `libsodium` under the hood). See
+https://pynacl.readthedocs.io/en/latest/signing/
+
+The resulting signature is base32-encoded and included in the "tahoe
```

Comment:
The implementation uses base64.  Either of these seems fine.  I'm not sure why we would pick base32 over base64 except for some kind of consistency with Tahoe-LAFS' choice of text-ish encoding function.